### PR TITLE
TortoiseSVN

### LINKED ISSUE #1120: tortoisesvn installer throws errors
I've tried installing tortoisesvn in regular and admin powershell, both with and without using sudo, but I keep getting an error. Here's the install output:

```
scoop install tortoisesvn
Installing 'tortoisesvn' (1.9.7) [64bit]
Loading TortoiseSVN-1.9.7.27907-x64-svn-1.9.7.msi from cache
Checking hash of TortoiseSVN-1.9.7.27907-x64-svn-1.9.7.msi ... ok.
Extracting TortoiseSVN.msi ... done.
Running installer... System.InvalidOperationException: This command cannot be run due to the error: The system cannot find the file specified.
   at System.Management.Automation.MshCommandRuntime.ThrowTerminatingError(ErrorRecord errorRecord)
Installation aborted. You might need to run 'scoop uninstall tortoisesvn' before trying again.
```
Please let me know if I can provide any further info to help debug.

### PR DESCRIPTION
**Pull request by se35710**
Registry keys from [here](https://superuser.com/questions/250260/manually-add-the-tortoisesvn-registry-information).

Should solve issue #[2466](https://github.com/lukesampson/scoop/issues/2466) in [scoop](https://github.com/lukesampson/scoop).

Also fixes #1120.

### CODE REVIEW

**Comment by se35710** [comment]
@rasa Like this? I still copy the reg files into the apps folder, since I have to replace in one of the files. Better have them all together for the end user, right?

[`vscode.json`](https://github.com/lukesampson/scoop-extras/blob/master/vscode.json) and its siblings also downloads from github, maybe we should update them too?

File: scripts/tortoisesvn/tortoisesvn-install-tools.reg [hunk 1]
```diff
@@ -0,0 +1,20 @@
+REGEDIT4
+
+[HKEY_CURRENT_USER\Software\TortoiseSVN\DiffTools]
+".doc"="wscript.exe \"C:\\TortoiseSVN\\Diff-Scripts\\diff-doc.js\" %base %mine //E:javascript"
```
Comment (rasa) [inline scripts/tortoisesvn/tortoisesvn-install-tools.reg]: Should these be %TSVNHOME% instead of the hardcoded C:\\TortoiseSVN?

Comment (se35710) [reply scripts/tortoisesvn/tortoisesvn-install-tools.reg]: Yes, I totally missed those, thanks. Fixed now,

**Comment by se35710** [comment]
Should be okay now, I think.

**Pull request merged**